etc/Makefile

```diff
@@ -30,6 +30,10 @@ clean: ## Make a clean source tree
 KERNELSPECS := kernelspecs_all kernelspecs_yarn kernelspecs_conductor kernelspecs_kubernetes kernelspecs_docker
 kernelspecs: $(KERNELSPECS)
 
+# Specify which version of docker images you would like to use in your kernelspecs in IMAGE_VERSION.
+# This value will be baked into the kernelspecs
+export IMAGE_VERSION:=$(TAG)
+
```

Comment:
I don't think the export is necessary.   Instead, I think you just need to use a double-dollar `$$` when referencing this variable from a make-defined function.

What is the purpose of IMAGE_VERSION?  If its to override the value of $TAG - which is the value of $VERSION, then I believe the declaration should be `IMAGE_VERSION?=$(TAG)`.  However, we need to be careful because the publish targets use $(TAG).  As a result, I think I'd prefer we remove IMAGE_VERSION and change the declaration of TAG to `TAG?=$(VERSION)`.  Then one could build/push their own set of custom-tagged images using:

`make TAG=myCustomTag dist docker-images publish-images`

Reply:
Removed IMAGE_VERSION, and set TAG to ?=$(VERSION)